README.md

```diff
@@ -23,6 +23,19 @@ Here's an example of a recommended `iframe` tag making full use of Nextframe's f
 <script>iFrameResize({autoSize: false}, "#nextframe")</script>
 ```
 
+## Configuring nginx
+You should add following lines to your Nextcloud config:
+```nginx
+server {
+  ...
+  add_header Content-Security-Policy "frame-ancestors git.lotigara.ru" always;
```

Comment:
Don't do that. The `Content-Security-Policy` header is already managed by Nextframe.

Reply:
Hmm... I didn't know that. I'm using YunoHost, which uses `more_set_headers` instead of official `add_header`.